orca-core/src/main/java/com/netflix/spinnaker/orca/ExecutionStatus.java

```diff
@@ -31,62 +30,62 @@
    * The task is still running and the {@code Task} may be re-executed in order
    * to continue.
    */
-    RUNNING(false, false),
```

Comment:
fixed some funky formatting in here only (was initially exploring adding a CANCELING state but backed that out)